I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 319, in script
    N "The two of you go to sit down at a table."
NameError: name 'last_pressed' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 319, in script
    N "The two of you go to sit down at a table."
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\ast.py", line 921, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\exports.py", line 1373, in say
    who(what, *args, **kwargs)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\character.py", line 1266, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\character.py", line 927, in do_display
    display_say(who,
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\character.py", line 666, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 3810, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 4 more times]
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 572, in visit_all
    for d in self.visit():
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\layout.py", line 1494, in visit
    self.update(self.last_st, self.last_at)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\layout.py", line 1505, in update
    raw_child, redraw = self.function(st, at, *self.args, **self.kwargs)
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\layout.py", line 1596, in condition_switch_show
    return condition_switch_pick(switch), None
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\display\layout.py", line 1587, in condition_switch_pick
    if renpy.python.py_eval_bytecode(code):
  File "C:\Users\Silve\Desktop\RenPy\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "<none>", line 1, in <module>
NameError: name 'last_pressed' is not defined

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
 
Sun Dec  4 18:13:00 2022
